src/component/panels/MoleculePanel.jsx

```diff
@@ -196,7 +196,7 @@ const MoleculePanel = memo(
         if (displayerMode === '1D' && ranges && ranges.values) {
           return ranges.values;
         } else if (displayerMode === '2D' && zones && zones.values) {
-          return ranges.values;
+          return zones.values;
```

Comment:
can you use DISPLAYER_MODE constant inside if letteral value 1d and 2d, you can import it from 
import { DISPLAYER_MODE } from './reducer/core/Constants';